pages/linux/pamac.md

```diff
@@ -1,6 +1,8 @@
 # pamac
 
 > A command line utility for the GUI package manager pamac.
+> If AUR is not enabled by default, add `--aur` after a command.
```

Comment:
:thinking: is not available for every subcommand only `search`, `info` and `checkupdates`. https://gitlab.manjaro.org/applications/pamac/-/blob/master/data/doc/pamac.8.asciidoc

Reply:
Hmm yeah, and `upgrade` and `update`

Reply:
How about "If you need the AUR package(s), add --aur after a command."?

Reply:
Or "If you don't see the AUR package(s), add --aur after a command"

Reply:
Hrm, if this doesn't work for all the subcommands mentioned in the tldr page, then we should probably remove this.

Reply:
Maybe this could work?
"If you can't see the AUR packages, enable it in the GUI version or in /etc/pamac.conf"

Reply:
*you need

Reply:
Yeah, I think is a better tip

Reply:
Changed the tip.